code-challenges.test.js

```diff
@@ -56,4 +169,35 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("vowelTester1", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+        expect(vowelTester1).toEqual([1])
+
+describe("vowelTester2", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+        expect(vowelTester1).toEqual([0])
+
+        describe("vowelTester3", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+        expect(vowelTester1).toEqual([3])
+    
+    
 // b) Create the function that makes the test pass.
+
+//Psuedocode
+//Input - vowelTester1, vowelTester2, vowelTester3,
+//Output - the index position of the first vowel in each string
+    // create a variable that searches for the vowels (a,e,i,o,u)
+    // use the .indexof built in method to index the variable search
+
+const findVowel = (string) => {
+    const search = ("a","e","i","o","u")
+    const firstVowel = vowelTester1.indexOf(search)
+
+}
+console.log(findVowel(vowelTester1))
+console.log(findVowel(vowelTester2))
+console.log(findVowel(vowelTester3))
+
+// Test Failed 
+// Results came back undefined
```

Comment:
Undefined will often point to no return inside a function.  There may be a few other things to consider here as well too.  The indexOf method only accepts one argument of a string, however, the search variable has multiple strings which if console logged would only evaluate to "u".  Consider using iteration here.